playbooks/Operationalizing/persistent_storage.adoc

```diff
@@ -0,0 +1,111 @@
+= Using Persistent Storage in OSEv3
+Øystein Bedin <bedin@redhat.com>
+v1.0, 2015-10-27
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
+:toc: macro
+:toc-title:
+
+Persistent Storage can be used for multiple aspects of OSEv3, including for applications to keep state, for databases, etc. In OSEv3, setting up a persistent storage consists of 2 steps - setting up a volume and a claim. The configuration of the persistent storage volume is a task handled by the system administrator, while the claim is tied to the deployment of a pod. 
+
+The steps taken to set up a persistent volume, and a corresponding volume claim, are not directly tied to each other. However, as the below examples shows, labels can be used to correlate a claim with a volume. 
+
+More information about the persistent storage architecture in OSEv3 can be found https://docs.openshift.com/enterprise/3.0/architecture/additional_concepts/storage.html[here].
+
+toc::[]
+
+== Setting up a Persistent Volume
+
+With version 3.0.x of OpenShift Enterprise, the supported storage solution is based on NFS. 
+
+Below is an example of a Persistent Volume tied to the nfs share 192.168.25.55:/mnt/nfs1
+
+----
+[root@master1 ~]# cat pv_nfs1.json  
+{ 
+  "apiVersion": "v1", 
+  "kind": "PersistentVolume", 
+  "metadata": { 
+    "name": "registry", 
+    "labels": { 
+      "purpose": "registry" 
+    } 
+  }, 
+  "spec": { 
+    "capacity": { 
+        "storage": "20Gi" 
+    }, 
+    "accessModes": [ "ReadWriteOnce" ], 
+    "nfs": { 
+        "path": "/mnt/nfs1", 
+        "server": "192.168.25.55" 
+    }, 
+    "persistentVolumeReclaimPolicy": "Retain" 
+  } 
+} 
+----
+
+More information can be found https://docs.openshift.com/enterprise/3.0/admin_guide/persistent_storage_nfs.html[here].
+
+== Setting up a Persistent Volume Claim
+
+In order for a project to use as volume, a claim needs to be created. This is how OSEv3 knows to assign a volume, with matching properties, to a project. The project can then assign the claim to a pod. 
```

Comment:
I think that things read more better in paragraph form when we say just "OpenShift" rather than OSEv3. I think we should use the full OpenShift Enterprise 3 in headings, and just "OpenShift" in paragraphs. Thoughts? /cc @trevorquinn